tests/selectedtests/app/controllers/test_project_task_mappings_controller.py

```diff
@@ -85,108 +87,67 @@ def test_POST_work_item_inserted(
         build_variant_regex="!.*",
     )
 
-    response = app_client.post(
-        f"/projects/{project}/task-mappings",
-        data=json.dumps(test_params),
-        content_type="application/json",
-    )
+    response = app_client.post(f"/projects/{project}/task-mappings", json=test_params)
     assert response.status_code == 200
-    assert response.get_json()["custom"] == f"Work item added for project '{project}'"
+    assert response.json()["custom"] == f"Work item added for project '{project}'"
 
 
 @patch(ns("ProjectTaskMappingWorkItem"))
-@patch(ns("get_evg_project"))
-def test_POST_work_item_inserted_with_incorrect_params(
```

Comment:
Why remove this test?

Reply:
FastAPI coerces `3` to become `"3"` since the schema says it needs to be a string. So this won't succeed to fail.

Reply:
Well could you keep the test and change the failure to whatever the FastAPI failure response is?

Reply:
FastAPI doesn't fail, it just coerces 3 to "3" --> which is a valid string and can be used as a regex pattern.

Reply:
Oh I see now, makes sense